frontend/src/stores/user.js

```diff
@@ -0,0 +1,25 @@
+import { computed, ref } from 'vue';
+import { defineStore } from 'pinia';
+import userService from '@/api/users';
+import { useStorage } from '@vueuse/core';
+
+export const useUserStore = defineStore('user', () => {
+  const user = ref(useStorage('user', {}));
+  const token = ref(useStorage('token', ''));
```

Comment:
I am new to `useStorage` and `@vueuse/core` but looking at its [documentation](https://vueuse.org/core/usestorage/#demo) and [code examples](https://github.com/vueuse/vueuse/blob/main/packages/core/useStorage/demo.vue), the returned values seem to be reactive without calling `ref`.